pipeline/transformer/transformer_tei/src/test/scala/weco/pipeline/transformer/tei/TeiDataParserTest.scala

```diff
@@ -0,0 +1,60 @@
+package weco.pipeline.transformer.tei
+
+import org.scalatest.funspec.AnyFunSpec
+import org.scalatest.matchers.should.Matchers
+import weco.pipeline.transformer.tei.fixtures.TeiGenerators
+
+class TeiDataParserTest extends AnyFunSpec with Matchers with TeiGenerators{
+  val id = "manuscript_15651"
+
+  it("parses a tei xml and returns TeiData"){
+    val description = "a manuscript about stuff"
+    TeiDataParser.parse(TeiXml(id, teiXml(id = id, summary = Some(summary(description))).toString()).right.get) shouldBe Right(TeiData(id, Some(description), None))
+  }
+  it("parses a tei xml and returns TeiData with bNumber"){
+  val bnumber = "b1234567"
```

Comment:
Suggestion: there's something like SierraDataGenerators that will create Sierra b-numbers for you.